test/fixtures/kitchen/package.json

```diff
@@ -3,22 +3,18 @@
   "version": "1.0.0",
   "description": "An app we're using to test the library. ",
   "scripts": {
-    "check": "gts check",
-    "clean": "gts clean",
```

Comment:
this repo still uses transpiled typescript with a `compile` script. i think we should still have these helper scripts: `check`, `clean`, `fix`. if not, what's the equivalency?